cli/crates/server/src/sql/001_add_owned_by.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE records ADD COLUMN owned_by TEXT;
```

Comment:
I'm sad sqlite doesn't support `IF NOT EXISTS` :'(

Reply:
It does support that

Reply:
Ah, it doesn't support it in this specific case :(
But it does work for other cases, we use it in most places

Reply:
@Finistere it does support it, but not for this specific case for some reason